grakn-factory/janus-factory/src/test/java/ai/grakn/factory/GraknTxJanusTest.java

```diff
@@ -56,15 +57,21 @@
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertThat;
 import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 public class GraknTxJanusTest extends JanusTestBase {
+    private final static GraknSession session = mock(GraknSession.class);
```

Comment:
The super-class also has a (private) mock `session`. I would use that one if you can.